SabaCenter/Model/Services/IHadithService.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  IHadithService.swift
+//  SabaCenter
+//
+//  Created by Mohsen Hosseini on 4/7/18.
+//  Copyright © 2018 Saba. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+public protocol IHadithService {
+    func getHadiths() -> Observable<[HadithItem]>
+    func getHadithOfTheDay() -> HadithItem
```

Comment:
Make this return an Observable<HadithItem>. That way it's asynchronous, which is necessary because all network calls are asynchronous.